src/app/app.module.ts

```diff
@@ -10,21 +10,21 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-import { HttpClientModule } from '@angular/common/http';
-import { Routes, RouterModule } from '@angular/router';
+import {BrowserModule} from '@angular/platform-browser';
```

Comment:
Can we please revert the spacing and ordering changes in this file? this should be unneeded (and counters convention)